src/katello/agent/katelloplugin.py

```diff
@@ -162,7 +167,7 @@ def update_settings():
     # the client certs
     ca_candidates = [os.path.join(ca_cert_dir, 'katello-default-ca.pem'),
                      os.path.join(ca_cert_dir, 'candlepin-local.pem'),
-                     rhsm_conf['rhsm']['repo_ca_cert'] % rhsm_conf['rhsm']]
+                     rhsm_conf['rhsm']['repo_ca_cert'] % {'ca_cert_dir': ca_cert_dir}]
```

Comment:
This code has changed a lot since I originally wrote it.  the whole `ca_candidates` thing is new to me and I don't understand the need for the complexity.  I'd like to help but have to defer to katello team on this.